ui/lib/core/stories/info-table-row.md

```diff
@@ -4,22 +4,24 @@
 `InfoTableRow` displays a label and a value in a table-row style manner. The component is responsive so
 that the value breaks under the label on smaller viewports.
 
+**Params**
 
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
 | value | <code>any</code> | <code></code> | The the data to be displayed - by default the content of the component will only show if there is a value. Also note that special handling is given to boolean values - they will render `Yes` for true and `No` for false. |
 | label | <code>string</code> | <code>null</code> | The display name for the value. |
+| helperText | <code>string</code> | <code>null</code> | Text to describe the value displayed beneath the label. |
 | alwaysRender | <code>Boolean</code> | <code>false</code> | Indicates if the component content should be always be rendered.  When false, the value of `value` will be used to determine if the component should render. |
 
 **Example**
   
 ```js
-<InfoTableRow @value={{5}} @label="TTL" />
+<InfoTableRow @value={{5}} @label="TTL" @helperText="Some description"/>
 ```
 
 **See**
 
 - [Uses of InfoTableRow](https://github.com/hashicorp/vault/search?l=Handlebars&q=InfoTableRow+OR+info-table-row)
-- [InfoTableRow Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/info-table-row.js)
+- [InfoTableRow Source Code](https://github.com/hashicorp/vault/blob/master/ui/lib/core/addon/components/info-table-row.js)
```

Comment:
These changes should no longer show after you merge in the latest on this branch.

Reply:
can you clarify what you mean here? i ran `git pull origin ui/replication-status-menu-discoverability` and the changes are still showing up.

Reply:
Got it.  Maybe I didn't do these changes.  I thought I had. Ignore.

Reply:
heh, alright. thanks!